### PR TITLE
Feature - Add Appointment Creation

### LINKED ISSUE #94: Feature - Add Command for Appointment Creation
This task will cover

- [ ] Adding the Appointments Model to codebase
- [ ] Saving and Loading of Appointments Model
- [ ] Adding Command for Appointment Creation

### PR DESCRIPTION
**Pull request by sharpstorm**
## Changes
This PR adds the basic skeleton for Appointments and the command for creating appointments. The created PR cannot be displayed in the application in this PR, but will be reflected in the data file written to disk

## Related Issues
- Closes #94

### CODE REVIEW

**Review by dandaandaaaaaan — COMMENTED** [review]
6/63 files checked, leaving this comment here so i know where i left off when i read this again tomorrow

**Comment by KwanHW** [comment]
17/63 files reviewed (at 8 Mar 2022 1:45am). Leaving this comment for further review

File: src/main/java/seedu/contax/storage/StorageManager.java [hunk 1]
```diff
@@ -71,8 +76,35 @@ public void saveAddressBook(ReadOnlyAddressBook addressBook) throws IOException
 
     @Override
     public void saveAddressBook(ReadOnlyAddressBook addressBook, Path filePath) throws IOException {
-        logger.fine("Attempting to write to data file: " + filePath);
+        logger.fine("Attempting to write to address book data file: " + filePath);
         addressBookStorage.saveAddressBook(addressBook, filePath);
     }
 
+    @Override
```
Comment (dandaandaaaaaan) [inline src/main/java/seedu/contax/storage/StorageManager.java:83]: minor nit, can follow the existing format and add a comment before these set of methods to split it

File: src/main/java/seedu/contax/storage/StorageManager.java [hunk 2]
```diff
@@ -71,8 +76,35 @@ public void saveAddressBook(ReadOnlyAddressBook addressBook) throws IOException
 
     @Override
     public void saveAddressBook(ReadOnlyAddressBook addressBook, Path filePath) throws IOException {
-        logger.fine("Attempting to write to data file: " + filePath);
+        logger.fine("Attempting to write to address book data file: " + filePath);
         addressBookStorage.saveAddressBook(addressBook, filePath);
     }
```
Comment (dandaandaaaaaan) [inline src/main/java/seedu/contax/storage/StorageManager.java:82]: minor nit, can follow the existing format and add a comment before these set of methods to split it
```suggestion
        // ================ Schedule methods ==============================
```

**Review by dandaandaaaaaan — APPROVED** [review]
lgtm!
tried to read as much as possible think everything looks okay, didn't do any testing on my own but i think its good to merge in. if any bugs come up we can fix later

File: src/main/java/seedu/contax/storage/StorageManager.java (hunk 2 above)
Comment (sharpstorm) [reply src/main/java/seedu/contax/storage/StorageManager.java:82]: Fixed in commit a0ce0f6.

**Review by KwanHW — APPROVED** [review]
LGTM

**Pull request merged**